src/Dafda.v3.ncrunchsolution

```diff
@@ -0,0 +1,5 @@
+﻿<SolutionConfiguration>
```

Comment:
Any chance we can not have config files from everyone's tooling ending up in this repo? 🙏

Reply:
yup, thanks :)